narwhals/dataframe.py

```diff
@@ -582,9 +582,9 @@ def write_parquet(self, file: str | Path | BytesIO) -> Any:
 
             We can then pass either pandas, Polars or PyArrow to `func`:
 
-            >>> func(df_pd)
-            >>> func(df_pl)
-            >>> func(df_pa)
+            >>> func(df_pd)  # doctest:+SKIP
+            >>> func(df_pl)  # doctest:+SKIP
+            >>> func(df_pa)  # doctest:+SKIP
```

Comment:
these will create a `foo.parquet` locally. If we don't want to skip them, I'll add the file to `.gitignore`

what do you think? :)

Reply:
I would opt for skipping those! Nice catch 👌